lib/logger.js

```diff
@@ -8,15 +8,12 @@ function Logger() {
 	Object.keys(Logger.prototype).forEach(function (key) {
 		this[key] = this[key].bind(this);
 	}, this);
+
+	this.reporter = reporter;
```

Comment:
I don't think you need `reporter.api = api` if you are passing it to `tapReporter()`.